.azure-pipelines/client.test.live.yml

```diff
@@ -52,7 +52,16 @@ jobs:
 
       # To get realtime logging of test progress in Azure Pipelines, we need to use "script" instead of "Npm@1",
       # since "Npm@1" currently buffers output (https://github.com/Microsoft/azure-pipelines-tasks/issues/8171).
+      - script: npm run live-test-client-$(Package)
+        condition: or(eq(variables['Package'], 'storage-blob'), eq(variables['Package'], 'storage-file'), eq(variables['Package'], 'storage-queue'))
```

Comment:
I'd strongly prefer to keep the pipeline definition simpler, and move this complexity into the ServiceBus and/or root project.json.

Reply:
Removed the condition